components/brave_ads/core/internal/conversions/queue/conversion_queue_database_table.cc

```diff
@@ -139,7 +139,11 @@ void GetCallback(GetConversionQueueCallback callback,
   for (const auto& record : command_response->result->get_records()) {
     const ConversionQueueItemInfo conversion_queue_item =
         GetFromRecord(&*record);
-    conversion_queue_items.push_back(conversion_queue_item);
+    // TODO(https://github.com/brave/brave-browser/issues/33239): Validate all
+    // Brave Ads data when loading from database
+    if (conversion_queue_item.IsValid()) {
```

Comment:
Do we know why the data was not valid, because we will throw away the conversion here?

Reply:
As discussed in DM we should add a DumpWithoutCrashing to understand why the conversion is invalid and uplift to 1.58.x

Reply:
conversions remain in the database, so we will be able to process them later